src/Carousel/Carousel.xml

```diff
@@ -0,0 +1,194 @@
+<?xml version="1.0" encoding="utf-8"?>
+<widget id="Carousel.widget.Carousel" needsEntityContext="true" xmlns="http://www.mendix.com/widget/1.0/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mendix.com/widget/1.0/ ../../xsd/widget.xsd">
```

Comment:
For the new widgets we want to get rid of these arbitrary and strange ids. Please use reverse domains, so `com.mendix.widget.Carousel`. That way it'll never conflict with carousel widgets from other companies as well.